dfsclient/src/main/java/org/hpccsystems/dfs/client/RowServiceInputStream.java

```diff
@@ -76,7 +77,7 @@
      * @param rd
      *            the JSON definition for the read input and output
      */
-    public RowServiceInputStream(DataPartition dp, FieldDef rd, FieldDef pRd) throws Exception
+    public RowServiceInputStream(DataPartition dp, FieldDef rd, FieldDef pRd, int connectTimeout) throws Exception
```

Comment:
we might need to provide RowServiceInputStream(DataPartition dp, FieldDef rd, FieldDef pRd) and the new version for backwards compatibility